Model/Currency.hs

```diff
@@ -77,9 +83,19 @@ instance ToMarkup Milray where
 
 instance Read Milray where
     readsPrec p ('$':'-':s) = map (\ (Milray i, rest) -> (Milray $ -i, rest)) $ readsPrec p ('$':s)
-    readsPrec _ ('$':s) = let (ipart, more) = span (`elem` ",0123456789") s
+    readsPrec _ ('$':s) = let (ipart, more) = 
+#if MIN_VERSION_base(4,8,0)
+                                span (`elem` (",0123456789" :: String)) s
+#else
+                                span (`elem` (",0123456789") s
+#endif
                               (fpart, rest) = case more of 
-                                                  '.' : more' -> span (`elem` "0123456789") more'
+                                                  '.' : more' -> 
+#if MIN_VERSION_base(4,8,0)
+                                                        span (`elem` ("0123456789" :: String)) more'
```

Comment:
This annotation doesn't look like it requires an extension -- am I missing something?